### ISSUE TITLE
[Vanilla][Reputation] Steamwheedle Cartel reputation

### ISSUE THREAD

**Issue by Galedrian** [dalaranwow/dalaran-wow#4809]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The reputation with the four goblin factions (Booty Bay, Gadgetganz, Ratchet and Everlook) is not lowering by killing the cities guards.
**Expected behaviour**: Tell us what should happen instead
Each guard killed should drop the reputation with the four factions and rising the reputation with the Bloodsail Pirates.
**Steps to reproduce the problem**:

1. Kill a guard in Booty Bay
2. Watch your reputation with Bloodsail Pirates rising
3. Watch your reputation with the four goblin factions NOT lowering.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

http://www.wowhead.com/quest=4621/avast-ye-admiral 
this page shows that the reputation works like that.
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Galedrian, 
I can't confirm the issue on ptr.
![image](https://user-images.githubusercontent.com/6797611/31585774-3913d5e0-b1d0-11e7-914a-d521b697501e.png)
